renderer/containers/Request.js

```diff
@@ -14,12 +16,15 @@ const mapStateToProps = state => ({
   isProcessing: state.invoice.invoiceLoading,
   payReq: state.invoice.invoice,
   invoice: invoiceSelectors.invoice(state),
+  willUseFallback: settingsSelectors.currentConfig(state).useAddressFallback,
```

Comment:
Shouldn't this be `.invoices.useAddressFallback`?

Reply:
it should